### DISCUSSION TITLE
Recomended Armbian version for soc s905x2

### DISCUSSION THREAD

**Discussion by xFall00** [ophub/amlogic-s9xxx-armbian#1773] [Amlogic]
Hello Guys. What Armbian version and image would you recommend?

I have a TESLA MediaTech X700 Pro with soc s905x2 in it. I would like to use it to play movies via kodi and play retro games e.g. via RetroArch.

Previously I used CoreELEC on an sd-card that is great for movies. Retro gaming only with wired Xbox One or Xbox Series controllers (xone driver for MS USB Xbox dongle did not work there). I would like to use my Xbox controllers wirelessly.

Thank you for your advice.

**Comment by xFall00** [comment]
Ok solved, see https://forum.armbian.com/topic/30699-recomended-armbian-version-for-soc-s905x2/

**Comment by ophub** [comment]
Which dtb and u-boot are used to boot your device?

**Reply by xFall00** [reply]
updated the reference threat
